var/spack/repos/builtin/packages/dftbplus/package.py

```diff
@@ -8,15 +8,29 @@
 from spack.package import *
 
 
-class Dftbplus(MakefilePackage):
+class Dftbplus(MakefilePackage, CMakePackage):
     """DFTB+ is an implementation of the
     Density Functional based Tight Binding (DFTB) method,
     containing many extensions to the original method."""
 
     homepage = "https://www.dftbplus.org"
-    url = "https://github.com/dftbplus/dftbplus/archive/19.1.tar.gz"
-
-    version("19.1", sha256="4d07f5c6102f06999d8cfdb1d17f5b59f9f2b804697f14b3bc562e3ea094b8a8")
+    # url = "https://github.com/dftbplus/dftbplus/archive/19.1.tar.gz"
```

Comment:
Maybe we should extend the comment an say why the tar balls do not work

Reply:
done